ryosuke-yamada/3dfractaldb
# [WIP] Multi gpu dynamic loading #6

# What

#6

# Changes

* Added:
    * dynamic_renderer.py - Dynamic renderer class that can load with DataLoader.
    * eglinfo.py - printing available GPUs for `use_gpus` list.

# How to test

* Execute dynamic_renderer.py
    * Check if images are correctly generated
    * Check execution time
